magicgui/widgets/_bases/ranged_widget.py

```diff
@@ -29,8 +29,7 @@ def __init__(
         self,
         min: Union[float, _Unset] = UNSET,
         max: Union[float, _Unset] = UNSET,
-        step: float = 1,
-        adaptive_step: bool = True,
+        step: Union[float, _Unset, None] = UNSET,
```

Comment:
Could you motivate me to why you add `None` and `_Unset` here? One of them should be enough.

Reply:
That's because min and max are also initialized with UNSET... I guess UNSET is for `value` only (because it might be set to None intentionally) so should we just change the defaults of min, max and step to None?